cmd/gpu-feature-discovery/main.go

```diff
@@ -86,11 +86,16 @@ func main() {
 			Value:   "/etc/kubernetes/node-feature-discovery/features.d/gfd",
 			EnvVars: []string{"GFD_OUTPUT_FILE"},
 		},
-		&cli.StringFlag{
+		&cli.StringSliceFlag{
 			Name:    "imex-nodes-config-file",
+			Aliases: []string{"imex-nodes-config-files"},
 			Usage:   "Path to the IMEX nodes config file. This file contains a list of IP addresses of the nodes in the IMEX domain.",
-			Value:   "/etc/nvidia-imex/nodes_config.cfg",
-			EnvVars: []string{"GFD_IMEX_NODES_CONFIG_FILE"},
+			Value: cli.NewStringSlice(
+				"/etc/nvidia-imex/nodes_config.cfg",
```

Comment:
I would prefer constructing these paths dynamically based on the `DRIVER_ROOT` instead of hardcoding them

Reply:
The container path will always be either `/driver-root` or `/host`

Reply:
But that's an assumption we are injecting into the code. It could always change in the future. When we make this path dependent on a configurable value, it is most robust and future-proof IMO

Reply:
Could you expand on what you mean? These paths are container local and as such they are not the same as the driverRoot or (hostRoot for that matter).

Reply:
> The container path will always be either /driver-root or /host

This path exists because of the Kubernetes host mount, yes? It is external to the application, and it should only be passed in as an input to the application. Other than that, the application shouldn't have k8s details hardwired into it. It becomes a leaky abstraction IMO